boards/arm/sparkfun_thing_plus_nrf9160/sparkfun_thing_plus_nrf9160_defconfig

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_NRF91X=y
+CONFIG_SOC_NRF9160_SICA=y
+CONFIG_BOARD_SPARKFUN_THING_PLUS_NRF9160=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
```

Comment:
Can you also enable HW_STACK_PROTECTION =y here? It's default now in cortex-m boards.

Reply:
Added this to the `ns` and regular variant.